components/Header/Artist.tsx

```diff
@@ -46,29 +47,45 @@ const Artist = ({
 
   return (
     <button
-      className={`${
+      className={cn(
+        "py-2 w-full outline-none",
         isMini
-          ? `${isSelectedArtist && "w-fit rounded-full"} flex justify-center items-center`
-          : `flex gap-3 items-center px-2 text-sm rounded-md text-grey-dark ${isAnyArtistSelected ? 'hover:bg-grey-light-1' : ''} ${isSelectedArtist && "!bg-grey-light-1"}`
-      } py-2 w-full ${shouldHighlight ? 'z-50 relative' : ''}`}
+          ? [
+              "flex justify-center items-center",
+              isSelectedArtist && "w-fit rounded-full"
+            ]
+          : [
+              "flex gap-3 items-center px-2 text-sm rounded-md text-grey-dark",
+              isAnyArtistSelected && "hover:bg-grey-light-1",
+              isSelectedArtist && "!bg-primary/10"
+            ],
+        shouldHighlight && "z-50 relative"
+      )}
       type="button"
       onClick={handleClick}
       onMouseEnter={() => setIsHovered(true)}
       onMouseLeave={() => setIsHovered(false)}
     >
       <div className="relative">
         <div
-          className={`w-8 aspect-1/1 rounded-full overflow-hidden flex items-center justify-center ${isSelectedArtist && "shadow-[1px_1px_1px_1px_#E6E6E6]"} ${shouldHighlight ? 'brightness-110 shadow-md ring-1 ring-white/30' : ''}`}
+          className={cn(
+            "w-8 aspect-1/1 rounded-full overflow-hidden flex items-center justify-center p-0.5",
+            isSelectedArtist && "shadow-[1px_1px_1px_1px_#E6E6E6] min-w-8 min-h-8 border-2 border-primary box-content",
+            shouldHighlight && "brightness-110 shadow-md ring-1 ring-white/30"
+          )}
         >
```

Comment:
Classes are the same as earlier, but these are now arranged using tailwind-merge cn util function.

Addition:

Added: border-2 border-primary box-content (to fix image shrink issue), p-0.5 (to add minor gap between avatar image and rounded border)